pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -791,7 +791,7 @@ def construct_trials(seg_files, seg_dict, ifos, slide_dict, vetoes):
     # Separate segments
     trial_time = abs(segs['on'])
 
-    for slide_id in range(len(slide_dict)):
+    for slide_id, _ in enumerate(slide_dict):
```

Comment:
I made the change locally and regenerated the usual test page [here](https://ldas-jobs.ligo.caltech.edu/~francesco.pannarale/LVC/py3_pygrb_webpage_2/).  If I were to push the changes, it would expand this PR, but you did indirectly raise a valid point: right now `slide_dict` is not a dictionary!

Reply:
If you can go ahead and push the additional change, since it's been tested, then I'll quickly look at it.